.ddev/config.yaml

```diff
@@ -1,33 +1,31 @@
 name: tea
 type: php
 docroot: ""
-php_version: "8.1"
+php_version: "8.3"
 webserver_type: nginx-fpm
 router_http_port: "8081"
 router_https_port: "4434"
 xdebug_enabled: false
 additional_hostnames: []
 additional_fqdns: []
 database:
-  type: mariadb
-  version: "10.5"
-nfs_mount_enabled: false
-mutagen_enabled: false
+    type: mariadb
+    version: "10.5"
 hooks:
-  post-start:
-  - exec: composer install --no-progress
-omit_containers: [dba, ddev-ssh-agent]
+    post-start:
+        - exec: composer install --no-progress
+omit_containers: [ddev-ssh-agent]
```

Comment:
Shouldn't we still omit the DBA container? Or is this related to the DDEV upgrade?

Reply:
Will fix, related to ddev upgrade, but it should be migrated also.